src/meta-srv/src/service/admin.rs

```diff
@@ -223,6 +245,222 @@ fn boxed(body: String) -> BoxBody {
         .boxed_unsync()
 }
 
+/// Expose admin HTTP endpoints as an Axum router for the main HTTP server.
+pub fn admin_axum_router(metasrv: Arc<Metasrv>) -> AxumRouter {
+    let node_lease_handler = Arc::new(NodeLeaseHandler {
+        meta_peer_client: metasrv.meta_peer_client().clone(),
+    });
+    let heartbeat_handler = Arc::new(HeartBeatHandler {
+        meta_peer_client: metasrv.meta_peer_client().clone(),
+    });
+    let leader_handler = Arc::new(LeaderHandler {
+        election: metasrv.election().cloned(),
+    });
+    let maintenance_handler = Arc::new(MaintenanceHandler {
+        manager: metasrv.runtime_switch_manager().clone(),
+    });
+    let procedure_handler = Arc::new(ProcedureManagerHandler {
+        manager: metasrv.runtime_switch_manager().clone(),
+    });
+
+    AxumRouter::new()
```

Comment:
nit: we can use [nest](https://docs.rs/axum/latest/axum/struct.Router.html#method.nest) function